packages/superset-ui-chart-controls/src/sections/sections.tsx

```diff
@@ -126,3 +126,5 @@ export const annotations: ControlPanelSectionConfig = {
     ],
   ],
 };
+
+export default {};
```

Comment:
What's this for?

Reply:
good catch. It should be useless. do not use default export here. I will remove it.